scheduler-jobs/deploy_data_specific_image

```diff
@@ -82,10 +82,14 @@ for ((i=0;i<$CONFIG_LIST_LENGTH;i++)); do
 
     echo "Extracting ${FULL_URL} to ${FULL_DEST_DIR}"
 
-    # Download the data from github and copy it to the destination directory
-    # TODO: Handle renamed or deleted notebooks, since this command only copies the content of the source_dir in the dest_dir
-    # For now, any other file found in the dest_dir are not handled and remain in its location.
-    svn export --force $FULL_URL $FULL_DEST_DIR
+    # Download the data from github into tmp directory
+    svn export --force $FULL_URL tmp
+
+    # Synchronize source_dir with dest_dir
+    # --delete : delete extraneous files from destination dirs
+    rsync -rtovcs --delete tmp/ $FULL_DEST_DIR/
+    # Remove tmp directory
+    rm -r tmp
```

Comment:
Are you sure just `-r` is enough?  I usually have to do `-rf` to delete directories, especially when it's not empty.

Reply:
`-r` : remove directories and their contents recursively
`-f` : ignore nonexistent files and arguments, never prompt

So the `-f` option would simply ignore nonexistent files and arguments, so in the case where you give to `rm` a path that does not exist, it will not break and prompt a message about not finding the directory or file. 
So in this case, since the folder will always be created before removing it, it should not prompt any error, but we can add it anyway to make sure it does not break the entire script. 
What do you think?

Reply:
Yes, adding `f` is safer, and has no side effects.